data/mods/ssb/moves.js

```diff
@@ -1018,6 +1018,32 @@ let BattleMovedex = {
 		target: "normal",
 		type: "Psychic",
 	},
+	// Elgino
+	roughsnuggle: {
+		accuracy: 100,
+		basePower: 90,
+		category: "Physical",
+		desc: "Power doubles if the target is not fully evolved.",
+		shortDesc: "Power doubles if the target is NFE.",
+		id: "roughsnuggle",
+		name: "Rough Snuggle",
+		isNonstandard: "Custom",
+		pp: 10,
+		priority: 0,
+		flags: {protect: 1, mirror: 1},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, 'Let\'s Snuggle Forever', target);
```

Comment:
Just like with all other custom moves, you need to cancel the default animation first.
```javascript
onTryMove() {
	this.attrLastMove('[still]');
},
```